.github/workflows/antsibull-build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             antsibull_changelog_ref: 0.14.0
             antsibull_core_ref: stable-1
           - name: Ansible 8 with ansible-core devel
-            options: '-e antsibull_ansible_version=8.99.0 -e antsibull_ansible_git_version=devel'
+            options: '-e antsibull_ansible_version=8.99.0 -e antsibull_ansible_git_version=stable-2.15'
```

Comment:
IMO the "latest" one should always be 'devel'. This isn't the ansible-build-data repo which is about testing actual releases, this is more for testing the latest stuff we have. (Also the name still says 'ansible-core devel'.)

Reply:
Makes sense to me. I guess https://github.com/ansible-community/ansible-build-data/blob/58d821a4fe52b5effbe1a87ef04231623716609f/.github/workflows/antsibull-build.yml#L23 is what needs to be changed.